### PR TITLE
SAK-48662 wiki: fixed new comment popup

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-48662

### CODE REVIEW

File: rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/view.vm [hunk 1]
```diff
@@ -83,7 +83,7 @@
 				
 				#set( $value= ["#formatDisplayName(${currentRWikiObject.user})","#formatDateTime($currentRWikiObject.version)"])
 				${rlb.getFormattedMessage("jsp_last_modified_by", $value.toArray())}
-				<a id="toCommentsLink" href="#tocomments" style="display:none" style="padding:0 1em 0 0">${rlb.jsp_comment_gotolink} <span></span></a>
+				<buttona id="toCommentsLink" href="#tocomments" style="display:none" style="padding:0 1em 0 0">${rlb.jsp_comment_gotolink} <span></span></a>
```
Comment (jesusmmp) [inline rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/view.vm:86]: buttona?

File: rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/macros.vm [hunk 2]
```diff
@@ -219,7 +219,7 @@ ${util.formatDateTime($date,$request)}
 					<span id="totalComments" class="textPanelFooter"></span>
 					#if (${renderBean.canEdit} || ${renderBean.canCreate})
 						<span class="commentsToolBar">
-							<a href="#" id="createCommentLink" onclick="ajaxRefPopup(this,'${util.escapeHtml(${renderBean.newCommentURL})}',0); return false;" >${rlb.jsp_addcomment}</a>
+							<a href="#" id="createCommentLink2" onclick="newEditComment(this, '${util.escapeHtml(${renderBean.newCommentURL})}', false); return false;" >${rlb.jsp_addcomment}</a>
```
Comment (jesusmmp) [inline rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/macros.vm:222]: Is there another id with createCommentLink?

I can see only CSS for
  #createCommentLink {
      background: url(/library/image/silk/comment_add.png) center left no-repeat;
      padding:3em 1.6em;
  }

but not for createCommentLink2

Is this OK?

File: rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/view.vm (hunk 1 above)
Comment (adrianfish) [reply rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/view.vm:86]: lol. Well spotted. It's a new webcomponent called buttona

Comment (adrianfish) [inline rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/view.vm:86]:
```suggestion
				<a id="toCommentsLink" href="#tocomments" style="display:none" style="padding:0 1em 0 0">${rlb.jsp_comment_gotolink} <span></span></a>
```

File: rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/macros.vm (hunk 2 above)
Comment (adrianfish) [reply rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/macros.vm:222]: I'm glad you spotted this. I think I made a mistake when I pushed the branch. Thanks.

**Review by jesusmmp — COMMENTED** [review]
@adrianfish Autosave has been removed?

**Comment by adrianfish** [comment]
@jesusmmp Yes, it has been removed, but in a separate commit. It wasn't working, anyway. It was based on Flash and flash hasn't been in browsers since late 2020.

**Comment by jesusmmp** [comment]
> @jesusmmp Yes, it has been removed, but in a separate commit. It wasn't working, anyway. It was based on Flash and flash hasn't been in browsers since late 2020.

Great!. Although I think a different ticket would have been appropriate for this purpose.

**Review by jesusmmp — APPROVED** [review]
LGTM!

**Comment by jesusmmp** [comment]
> @jesusmmp We could still do that in another ticket. That's why I put it in a separate commit - I was a bit unsure myself.

For my part this time we can leave it this way. Please keep this in mind for next time. Thanks!

**Pull request merged**